src/katgpucbf/xbgpu/main.py

```diff
@@ -237,6 +307,11 @@ def make_engine(context: AbstractContext, args: argparse.Namespace) -> tuple[XBE
         monitor = NullMonitor()
 
     logger.info("Initialising XB-Engine on %s", context.device.name)
+    xengine = XOutput(
+        name="xengine",
+        dst=args.dst,
+        channels_per_substream=args.channels_per_substream,
+    )
```

Comment:
My sense of aesthetics almost wants the user to have to input
```
--corrprod=name=steven,dst=239.0.0.1
```
corresponding to the `--beam` argument(s). That way you bind the `dst` to the output quite nicely.

(Or some other word, I can't currently think of a better way to refer to the correlation products.)

Reply:
I agree with this. In fgpu I started out by keeping backwards compatibility in the first PR and later updating it to take the wideband config in `--wideband`, but doing that involved a lot of incremental refactoring and I think it's going to be cleaner to bite the bullet now.